webapp/sources/rudder/rudder-core/src/test/resources/node-facts/node-001-machine-physical-v2.json

```diff
@@ -26,8 +26,8 @@
     "user": "machine-user",
     "version": "3.15.3",
     "securityToken": {
-      "kind": "certificate",
-      "token": "-----BEGIN CERTIFICATE-----\\nMIIFqzCCA5OgAwIBAgIUJO5/m8PZResSZZjA8L9zSRvkaXkwDQYJKoZIhvcNAQEL\\nBQAwNjE0MDIGCgmSJomT8ixkAQEMJDkzMGRjY2IwLTViMDYtNDM4NS1hYjdhLTgw\\nYTdlNTZlYjk2OTAeFw0yNDA1MDIxMzQ2NTRaFw0zNDA0MzAxMzQ2NTRaMDYxNDAy\\nBgoJkiaJk/IsZAEBDCQ5MzBkY2NiMC01YjA2LTQzODUtYWI3YS04MGE3ZTU2ZWI5\\nNjkwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQC6vbgDy39n/TGO0B/4\\n+Q+cd0c2IrIsfVYqyOnDn52sf4zGfSBfcmEtdvJSIb9Qp2ATGtKUh162/SOHOj5R\\nzqKW5tE4Danj8dxs2V0AUl5/rvlvl4RqN22gyQZirJ7Z5WtAPCAKmH5oiG/Igblr\\n26X2FmiPeei08caZNh48OykAa50sNdsFE2VkZr0mI/hltXxgjEKxPiQBvm0gTvQY\\nug/qgfxXwqQzgIHCmryB4IPIMSY/8LDNnY5BVk8vFlW44M7+m/U3ALPlYk+6kax0\\ngay9hNuMQlEPj+22OjMetBt7tPQd/VhZTL9RNLj736qLuCy8wV+60O4QrWXnkgK0\\nVS9IOgU+MP6mqT5Xrx18QCr0yTRgfaGXQz8XcAC/3fbdPMc4zhQwsR+2gHm8XjXi\\n0iXkYDK/TgkjvkTAePdZbQrjADnTZF/eQaxt7r9V1TEFJAvMsHorMUgurXW9JO96\\nXX+hT6Qp7BKFepOX/fCkped1HV1SUod7N0L/xkD5oZVhz4a1Z5ImEdRMRVgtm2+I\\nw5ng1mujcaa/2iHEmenjK6mLcOh+IXrTEXRoV5JBGmTxEYvdKn9Lvv0LJUwc3WFs\\nBR6/AHBAhvd0U2IHMuxki3AaTM+A6atKUo4Hk2MC967Ecz3Tg06YIoZxJPqo2Qij\\n3ib2izzRFuzuVuECO9DNI51OewIDAQABo4GwMIGtMAwGA1UdEwQFMAMBAf8wHQYD\\nVR0OBBYEFIjIpuhURlIay8ffm0w4enOJvoeHMHEGA1UdIwRqMGiAFIjIpuhURlIa\\ny8ffm0w4enOJvoeHoTqkODA2MTQwMgYKCZImiZPyLGQBAQwkOTMwZGNjYjAtNWIw\\nNi00Mzg1LWFiN2EtODBhN2U1NmViOTY5ghQk7n+bw9lF6xJlmMDwv3NJG+RpeTAL\\nBgNVHQ8EBAMCArwwDQYJKoZIhvcNAQELBQADggIBAJRQHHK9RD6ByGOFJ+8w3yH0\\n8QJ78uZe3K+N2dWwbLo6XSV9nvtkfyuUxsaBTTgXSmvkSlGNrCFnWW5vaea/R1Rq\\nw4zGlyS4UzBD31rc2kKl5/yIljYiufYsbMmYqh8Sy/U75msp4inRzVlyxX1FtkgW\\nDYr8nrcHbD94xuT7HTFX8HFAUOsGhEGDYXGvY94iJ4Jx7p5bWiVDbIME7Bf72+Ql\\ncWUapVCXVbuqWW6qw048EGcwjmMXJXNJgBth39yX2cEWklkSENuYwSLAloetUmYY\\nuSl6Y20pTKjSdUTBGIu0uN+0740LuiMlQ1nbdrBBasgLxH/1TfAp927zuQCPKANA\\n1O7OuhrF3gwLOVDjkycAhQb9nynZgpexmFkOMklXVYNWKp6s88thB3gTRCT01GFx\\ntY5WzobFShmv6AEmUWYutFG5k7fwNKGI6VEUVoQlNiHeDW4VV4AFrfK+/vNnJepf\\nqbSyRhS8QBgqt4KbwMdarVEZTvVrc6JhfnTSzXig6dO8bVA1EVLp0xyCd5eoplEV\\nHzhAz2MOrfOUmBjo2HcHA4tpcuE+ImNWuQiDfTdjqkZWRO5pBwbvnVLOGCqtaH1C\\nsGM8wv+QA1Y2NhgzADIb6u4PjXcBy6hqyAFKtvJ2J1pBL1Vtu4rTf9zhZ62HSqBd\\nRlYSlie1YMo1Yy2jzZhP\\n-----END CERTIFICATE-----"
```

Comment:
we could have read kind/token, but the test then serialize again, and the preference goes to type/value (to keep what is in LDAP), and to the back comparison fails.